documentation/synthetic_tokens/tutorials/prerequisites.md

```diff
@@ -24,7 +24,14 @@ If everything worked, we should see the line "> Compiled successfully using:" in
 
 1. Install the [Ganache UI](https://truffleframework.com/ganache).
 2. Run Ganache on localhost port `9545` (use the above links for instructions on how to do this).
-3. Note that to deploy new `ExpiringMultiPartyCreator`, the main factory used in the priceless financial contract, you must set your gas limit to at least 9 million in Ganache. To do this click the gear icon at the top right of Ganache, then click "CHAIN" and set the "GAS LIMIT" field to `10000000`. Then click the "RESTART" button top right. This will enable your Ganache instance to deploy larger contracts.
+3. Under "Accounts & Keys", set the "Account Default Balance" to `1000000`. This will ensure that you have enough local testnet ETH to run through tutorials.
+4. Note that to deploy new `ExpiringMultiPartyCreator`, the main factory used in the priceless financial contract, you must set your gas limit to at least 9 million in Ganache. To do this click the gear icon at the top right of Ganache, then click "CHAIN" and set the "GAS LIMIT" field to `10000000`. Then click the "RESTART" button top right. This will enable your Ganache instance to deploy larger contracts.
+
+You can also make all of these changes using the Ganache CLI tool with the following command:
```

Comment:
WDYT about saying "You can also run the Ganache CLI tool with identical settings as follows:" ?

Reply:
Sounds good - I'll rephrase the sentence.